tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/annotations/AnnotationAttachmentSymbolsTest.java

```diff
@@ -394,6 +394,19 @@ public void testConstAnnots() {
         assertAttachmentSymbol(attachmentsF4.get(0), "v29", true, "increment", -2L);
     }
 
+    @Test
+    public void testAnnotWithServiceRemoteMethodAttachmentPoint() {
+        BLangFunction function = getFunction("ServiceClass.serviceRemoteFn1");
+        List<? extends AnnotationAttachmentSymbol> attachments = function.symbol.getAnnotations();
+        Assert.assertEquals(attachments.size(), 2);
+
+        AnnotationAttachmentSymbol v31 = attachments.get(0);
+        Assert.assertFalse(v31.isConstAnnotation());
+        assertAttachmentSymbol(v31, "v31");
```

Comment:
Can't we use the same assertion method as the below assertion to assert the values in `v31`?

```java
assertAttachmentSymbol(v31, "v31", v31.isConstAnnotation(), null, null);
```

Reply:
This wouldn't test what we want to test, would it? We want to assert that `v31` is not a const annotation. 

```java
    private void assertAttachmentSymbol(AnnotationAttachmentSymbol attachmentSymbol, 
                                        String annotationTag,
                                        boolean constAnnot, 
                                        String key, 
                                        Object value) {
        ...
        Assert.assertEquals(annotationAttachmentSymbol.isConstAnnotation(), constAnnot);
        ...
    }
```

When we pass `v31.isConstAnnotation()` as the argument, the method asserts it against `annotationAttachmentSymbol.isConstAnnotation()`, which will always be true, since they are the same.

`org.ballerinalang.test.annotations.AnnotationAttachmentSymbolsTest#assertAttachmentSymbol(org.ballerinalang.model.symbols.AnnotationAttachmentSymbol, java.lang.String)` that is called already does

```java
assertAttachmentSymbol(attachmentSymbol, annotationTag, false, null, null)
```

which is what we want. Will get rid of L404 though, since it duplicates the false check.

Reply:
Make sense. Thanks!